Daspren/daspren-parad/ingest/parser.yml

```
@@ -0,0 +1,36 @@
+name: daspren-parad
+pipeline:
+  - name: json
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+        output_field: event
+  - name: set_recommended
+  - name: set_fields
+
+stages:
+  set_recommended:
+    actions:
+      - set:
+          observer.vendor: "Daspren"
+          observer.product: "Parad"
+          observer.type: "dlp-solution"
+  set_fields:
+    actions:
+      - set:
+          "@timestamp": "{{json.event.time}}"
+          process.name: "{{json.event.executable_basename}}"
+          process.executable: "{{json.event.executable}}"
+          process.pid: "{{json.event.pid}}"
+          process.parent.pid: "{{json.event.ppid}}"
+          daspren.process.hash.blake3: "{{json.event.hash}}"
```

According to the tests, file_opened.json's event contains information in the `event_data` property, while for the malicious_behaviour.json's message contains information at the root.
We suggest modifying these lines:
```suggestion
          process.name: "{{json.event.executable_basename or json.event.event_data.executable_basename}}"
          process.executable: "{{json.event.executable or json.event.event_data.executable}}"
          process.pid: "{{json.event.pid or json.event.event_data.pid}}"
          process.parent.pid: "{{json.event.ppid or json.event.event_data.ppid}}"
          daspren.process.hash.blake3: "{{json.event.hash or json.event.event_data.hash}}"
```